### PR TITLE
feat(ui5-lib-ref): prompt choice fix

### PR DESCRIPTION
**Pull request by cianmSAP**
Fixes prompt choices for projects and libs
i18n fix
Read me correction

### CODE REVIEW

**Review by devinea — COMMENTED** [review]
Code and test updates look good
changeset 👍

**Review by devinea — APPROVED** [review]
Code and test updates look good
changeset 👍

**Review by longieirl — APPROVED** [review]
Changes look good.
Test coverage looks good.

File: packages/ui5-library-reference-inquirer/README.md [hunk 1]
```diff
@@ -49,7 +49,9 @@ export default class UI5LibraryReferenceGenerator extends Generator {
 
         try {
             await generate(this.answers.targetFolder, reuseLibConfigs);
-        } catch (e) {}
+        } catch (e) {
+            console.log(`Error adding reference ${e}`)
```
Comment (IainSAP) [inline packages/ui5-library-reference-inquirer/README.md:53]: Should be using either the generator logger or preferably the open ux tools logger.

Comment (longieirl) [reply packages/ui5-library-reference-inquirer/README.md:53]: I had made the same comment but reverted it as it was a README! And I guess its whoever is implementing the logic would be responsible for the logger?

**Pull request merged**